BUILD

```diff
@@ -239,11 +239,13 @@ da_haskell_repl(
     visibility = ["//visibility:public"],
     deps = [
         ":damlc",
+        "//compiler/damlc/stable-packages:generate-stable-package",
```

Comment:
I don’t really like this giant repl target  (I care much more about being able to start ghci reasonably fast and reasonable memory usage than being able to load everything into a single session) but given that I already don’t use it now, I’m happy for you to make this change if @associahedron who I think uses this is fine with this.

Reply:
I'd also prefer to keep `da-ghci` slim.

Reply:
my experience was that it didn't get any slower

Reply:
it's a couple of extra haskell files on top of the 170 already being watched